socorro/processor/processor_2015.py

```diff
@@ -158,19 +149,29 @@ def __init__(self, config, quit_check_callback=None):
         else:
             self.quit_check = lambda: False
 
-        # here we instantiate the rule sets and their rules.
         self.rule_system = OrderedDotDict()
-        for a_rule_set_name in config.rule_sets.names:
-            self.config.logger.debug(
-                'setting up rule set: %s',
-                a_rule_set_name
-            )
-            self.rule_system[a_rule_set_name] = (
-                config[a_rule_set_name].rule_system_class(
-                    config[a_rule_set_name],
-                    self.quit_check
+        self.rules = []
+
+        # here we instantiate the rule sets and their rules.
+        if 'transform_rules' not in config:
+            return  # no rules
```

Comment:
Given that we've still got `rule_sets` configurable, it's possible that the rule does not have a `transform_rules` name/namespace. I think I'd go back to iterating over the lists in `config.rule_sets`.

Reply:
Actually, I take that back--we should nix the configurability of `rule_sets`. There should be one rule set and it should be defined in code and not available in configuration.